qiskit_experiments/whole_backend/whole_backend.py

```diff
@@ -0,0 +1,79 @@
+"""
+Functions to build composite experiments for entire backends
+"""
+
+from copy import deepcopy
+from qiskit.circuit import QuantumCircuit, Qubit, Clbit
+from qiskit_experiments.framework import (
+    BatchExperiment,
+    ParallelExperiment,
+    BaseExperiment,
+)
+
+
+class BasicExperiment(BaseExperiment):
+    """
+    Basic atmoic experiment that mimics the template experiment,
+    but uses a pre-prepared transpiled circuit
```

Comment:
`__init.py__` needs to be formatted for generating API docs, and this docstring needs more information on how this works and how to use it. Including the example in your PR text would be helpful.